frontend/src/api/matching-service/MatchingService.ts

```diff
@@ -18,14 +18,14 @@ let previousStartMatchingData : any = null;
 /**
  * An async function for sending a start matching request to the backend.
  * 
- * @param token The current user's token.
+ * @param id The current user's token.
```

Comment:
don't forget to change the documentation!

Reply:
oops okay